src/client/Microsoft.Identity.Client/IAccount.cs

```diff
@@ -32,6 +32,12 @@ public interface IAccount
         /// AccountId of the home account for the user. This uniquely identifies the user across AAD tenants.
         /// </summary>
         /// <remarks>Can be null, for example if this account was migrated to MSAL.NET from ADAL.NET v3's token cache</remarks>
-        AccountId HomeAccountId { get; } 
+        AccountId HomeAccountId { get; }
+
+        /// <summary>
+        /// The initial flow that established the account. For example, device code flow.
+        /// </summary>
+        /// <remarks>Can be null. Currently only device code flow updates this property with a valid string</remarks>
+        string AccountSource { get; }
```

Comment:
This is a breaking change. You can't add properties to an interface. Reverting this commit.

Reply:
IMO this should not be added to the public API.